### ISSUE TITLE
The new linker ld script doesn't seem to report bss size correctly

### ISSUE THREAD

**Issue by fredizzimo** [tmk/infinity_ergodox#4]
When building the current master branch of infinity_ergodox I got this linker output
   text    data     bss     dec     hex filename
  13181     496   65027   78704   13370 build/ch.elf

However, before the recent updates, I got this
   text    data     bss     dec     hex filename
  12633     496    4603   17732    4544 build/ch.elf

I checked the map file, and the according to my calculation only 3032 bytes seems to be used in the BSS section in the newest version, although it reports 65027. I'm not very familiar with ld scripts, and didn't have more time right now, so I haven't checked further.

I haven't actually tried this on my keyboard, as it seems like it would use more ram than available, if the output is correct.

I'm using the latest version of the the flabbergast kinetis ChibiOS branch.

**Comment by fredizzimo** [comment]
I investigated this further, and it's a know "feature" of ChibiOS, http://forum.chibios.org/phpbb/viewtopic.php?f=24&t=3104&p=23944

Basically the heap is now its own section, and therefore reported as a part of the BSS. I submitted a pull request to TMK_CORE to make the size report usable again https://github.com/tmk/tmk_core/pull/7. The section size output now uses the more verbose sysv format, which reports the BSS size correctly, and the heap as a separate entry.
